supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -382,6 +382,18 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[[inject]]
+==== image:images/yes.png[yes] inject (verb)
+*Description*: When data is _injected_, an object or function _receives_ other required objects or functions from external code instead of creating them internally; to populate application input fields with embedded control or command sequences. 
```

Comment:
```suggestion
*Description*: When data is _injected_, an object or function receives other required objects or functions from external code instead of creating them internally; to populate application input fields with embedded control or command sequences. 
```